### DISCUSSION TITLE
How to ignore bitwise or with RegexOptions

### DISCUSSION THREAD

**Discussion by AshleighAdams** [stryker-mutator/stryker-net#1494] [Q&A] [answered]
Hey, is it possible to ignore bitwise operations on RegexOptions?

I tried adding `System.Text.Regex.RegexOptions.op_BitwiseOr` to my `ignore-methods` options, but no dice.

**Comment by rouke-broersma** [comment, marked as answer]
Hi 👋 

The `ignore-methods` option is used to ignore mutations on method invocations and not operators. You can use the `[ExcludeFromCodeCoverageAttribute]`. This ignores the scope it is placed on from mutation testing. Does that help you?

**Reply by AshleighAdams** [reply]
Surely an operator is just syntactic sugar for a method call?

For now, I can work around using `[ExcludeFromCodeCoverage]` by switching from a private field to a private property, but I still want to mutation test the regex pattern later on, which will break this.

**Reply by rouke-broersma** [reply]
We do AST (syntax tree) mutations and not bytecode/IL mutations so we see the syntactic sugar unfortunately. Conceptually perhaps it makes sense to allow ignore-methods to work on operators as well, but currently that is not supported. You could create a feature request for this.

**Reply by AshleighAdams** [reply]
Ah, yeah, I see the trouble then---you won't know the fully qualified function it is syntactic sugar for either, not without a lot of work anyway.

I'll mark the `ExcludeFromCodeCoverage` solution as the answer then for now.
